### DISCUSSION TITLE
"No data available in table" after update docker image on unraid

### DISCUSSION THREAD

**Discussion by roukmoute** [dani-garcia/vaultwarden#2635] [Q&A] [answered]
After an update of vaultwarden docker image (2022.6.2) on my unraid machine, I can't connect anymore.

* Clients used: web vault
* SQLite

After connecting me on `/admin`, I see this on `Users` tab:

![image](https://user-images.githubusercontent.com/2140469/179995711-3ff690e6-e899-4711-b798-b2be9de623bb.png)

Where is stored my passwords with SQLite?

I've checked this:

```sh
/mnt/user/appdata/bitwarden# ls -la
total 424
drwxrwxrwx 1 nobody users     24 Jul 20 15:24 ./
drwxrwxrwx 1 nobody users   4096 Jul 20 15:15 ../
drwxr-xr-x 1 root   root       6 Jul 20 12:46 attachments/
-rw-r--r-- 1 root   root    1350 Jul 20 14:50 config.json
-rw-r--r-- 1 root   root  196608 Jul 20 15:21 db.sqlite3
-rw-r--r-- 1 root   root   32768 Jul 20 15:24 db.sqlite3-shm
-rw-r--r-- 1 root   root       0 Jul 20 15:24 db.sqlite3-wal
-rw-r--r-- 1 root   root  188416 Jul 20 14:51 db_20220720_125120.sqlite3
drwxr-xr-x 1 root   root       6 Jul 20 14:44 icon_cache/
-rw-r--r-- 1 root   root    1675 Oct 17  2021 rsa_key.pem
-rw-r--r-- 1 root   root     451 Oct 17  2021 rsa_key.pub.pem
drwxr-xr-x 1 root   root       6 Jul 20 12:46 sends/
drwxr-xr-x 1 root   root       6 Jul 20 12:46 tmp/
```

I'm really worried, because if a simple update broke all my passwords, I am struggling because I have no copy of theses passwords.

**Comment by BlackDex** [comment]
First, if you have any client still having the data visible, try to export that data as a json to preserve what you have right now.

**Comment by BlackDex** [comment]
Second. An update/upgrade of the container doesn't delete any data. There could be some database migrations, but that is about it.

I do see two sqlite3 file of which one seems to be a backup, and the other seems to be the current version, since it also has the `-shm` and `-wal` temp files there.

Those should contain all the data, and just in case, i would copy those files to an other safe place just in case.

Besides that, i would double check your docker config. It could be that unraid (which i have never used my self) created a new volume, which in turn generates an empty environment and no passwords/accounts.

Please check the volume location and if that matches.

**Comment by roukmoute** [comment, marked as answer]
Ok I've just seen my hidden `Storage` key.

Before:
![image](https://user-images.githubusercontent.com/2140469/180015916-2d5743c8-a0e1-41da-abc0-39fb8360c44f.png)

After:  
![image](https://user-images.githubusercontent.com/2140469/180015745-90969196-d2d0-49e7-a77c-f7f10909b3b2.png)

I have absolutely no idea why the update... update the value of this key.

So now it works, few...

Thanks a lot
